gemfiles/railsmaster.gemfile

```diff
@@ -3,5 +3,6 @@ source 'https://rubygems.org'
 gem 'arel', github: 'rails/arel'
 gem 'rails', github: 'rails/rails'
 gem 'sqlite3'
+gem 'rubocop-md'
```

Comment:
Let's add it to `gemspec` instead of adding to every gemfile

Reply:
It sounds good
https://github.com/palkan/clowne/pull/10/commits/50e783099369bbe86cbe75cf605744e6fe62bdd4